app/src/main/java/com/example/tmdb/screens/home/Home.kt

```diff
@@ -212,15 +235,42 @@ fun <T : Any> RowSectionItem(
 fun RowItem(
```

Comment:
Please revisit this whole method regarding this previous comment. you're using reflection in several situations here:

```kotlin
val getId = posterUrl?.javaClass?.getDeclaredField("id")
    getId?.isAccessible = true
```

```kotlin
viewModel.isAFavorite(getId?.get(posterUrl) as Int)
```

both are examples of reflection usage - please find them all and change them